### PR TITLE
Update "PHP Version" and "CodeSniffer" sections

### PR DESCRIPTION
**Pull request by DaazKu**
Update the Coding Standard section.

Update PHP Version section's information.
Link to the correct repository for code-sniffer standards.

### CODE REVIEW

File: content/developer/contributing/coding-standard.md [hunk 1]
```diff
@@ -20,22 +20,22 @@ We adopted this standard for all the usual reasons: our team is growing, and we
...
-We currently plan to move to PHP 5.5 after the fork for 2.3. These plans may change with little notice.
+For up to date information you can look at our [self-hosting-requirements](https://github.com/vanilla/vanilla/tree/master#self-hosting-requirements) section.
 
-### Check out CodeSniffer
+### Validating your PHP code with PHP_CodeSniffer
 
 CodeSniffer is a tool you can plug into your IDE to help you adhere to a coding standard like this one. This quick guide assumes you have both PEAR and homebrew with PHP 5.5 installed.
 
-Our standard is in [Vanilla's Addons repo](http://github.com/vanillaforums/addons), where you can submit issues and pull requests. Its contents will help you on your way.
+Our standard is in [Vanilla's Standars repo](https://github.com/vanilla/standards), where you can submit issues and pull requests. Its contents will help you on your way.
```
Comment (initvector) [inline content/developer/contributing/coding-standard.md]: > Standards

File: content/developer/contributing/coding-standard.md [hunk 2]
```diff
@@ -20,22 +20,22 @@ We adopted this standard for all the usual reasons: our team is growing, and we
...
+All code in `master` must work under PHP 5.6 but we strongly recommend using 7.0.
 
-All code in `release/2.2` must work under PHP 5.3.
+All code in `release/2.3` must work under PHP 5.4.
 
-We currently plan to move to PHP 5.5 after the fork for 2.3. These plans may change with little notice.
+For up to date information you can look at our [self-hosting-requirements](https://github.com/vanilla/vanilla/tree/master#self-hosting-requirements) section.
 
-### Check out CodeSniffer
+### Validating your PHP code with PHP_CodeSniffer
```
Comment (initvector) [inline content/developer/contributing/coding-standard.md]: Can this stay as "CodeSniffer"? It's "PHP_CodeSniffer" here, but "CodeSniffer" multiple times in the following lines.

File: content/developer/contributing/coding-standard.md [hunk 3]
```diff
@@ -20,22 +20,22 @@ We adopted this standard for all the usual reasons: our team is growing, and we
...
 ### PHP Version
 
-All code in `master` must work under PHP 5.4.
+All code in `master` must work under PHP 5.6 but we strongly recommend using 7.0.
 
-All code in `release/2.2` must work under PHP 5.3.
+All code in `release/2.3` must work under PHP 5.4.
 
-We currently plan to move to PHP 5.5 after the fork for 2.3. These plans may change with little notice.
+For up to date information you can look at our [self-hosting-requirements](https://github.com/vanilla/vanilla/tree/master#self-hosting-requirements) section.
```
Comment (initvector) [inline content/developer/contributing/coding-standard.md]: > up-to-date

File: content/developer/contributing/coding-standard.md [hunk 4]
```diff
@@ -20,22 +20,22 @@ We adopted this standard for all the usual reasons: our team is growing, and we
 
 ### PHP Version
 
-All code in `master` must work under PHP 5.4.
+All code in `master` must work under PHP 5.6 but we strongly recommend using 7.0.
```
Comment (initvector) [inline content/developer/contributing/coding-standard.md]: > ... PHP 5.6, but ...

File: content/developer/contributing/coding-standard.md (hunk 2 above)
Comment (DaazKu) [inline content/developer/contributing/coding-standard.md]: PHP_CodeSniffer is the official name.

Comment (initvector) [inline content/developer/contributing/coding-standard.md]: Then shouldn't it be PHP_CodeSniffer everywhere and not just in the section title?

Comment (DaazKu) [inline content/developer/contributing/coding-standard.md]: It's kind of ugly. People will know :P

Comment (DaazKu) [inline content/developer/contributing/coding-standard.md]: I felt bad not referencing the correct name at least once.

**Pull request merged**